### DISCUSSION TITLE
Accidentally clicked "always" on `commit all`. How do I revert the setting?

### DISCUSSION THREAD

**Discussion by dili021** [gitkraken/vscode-gitlens#2241] [Q&A]
Title says it all.

**Comment by eamodio** [comment]
I'm not sure what `commit all` you are talking about -- can you provide more details please? Thanks!

**Reply by dili021** [reply]
Sure, sorry about that. So, when you click on `commit` in the gitlens tab in vscode's side panel without any staged changes, you get a prompt to commit all changes. The given options are `no`, `yes`, `always`, maybe more, not sure. So, I accidentally clicked always, and now I don't get prompted anymore, clicking commit just commits everything. What I want is to revert this and get prompted again.

**Reply by d13** [reply]
The commit button is part of the Source Control view provided by VS Code, not GitLens. I'm not 100%, but I believe the "Smart Commit" settings control that.

Here's a couple settings that may want to look at:
- "Git: Enable Smart Commit" (`git.enableSmartCommit`)
- "Git: Smart Commit Changes" (`git.smartCommitChanges`)

**Reply by dili021** [reply]
Thank you!
